okio/src/commonMain/kotlin/okio/Path.kt

```diff
@@ -194,6 +194,16 @@ expect class Path internal constructor(bytes: ByteString) : Comparable<Path> {
    */
   operator fun div(child: Path): Path
 
+  /**
+   * Returns this path relative to [other].
```

Comment:
Gimme more docs....

This inverts the resolve operator, `/`. For any two paths `a` and `b` that have the same root, `(a / (b.relativeTo(a))) == b`

Reply:
Let’s also add that invariant to the test suite?! Might find bugs in this, or in resolve!